profile-widget/profilewidget2.cpp

```diff
@@ -1119,7 +1119,7 @@ void ProfileWidget2::updateVisibility()
 	decoModelParameters->setVisible(prefs.calcceiling);
 #ifndef SUBSURFACE_MOBILE
 	for (DiveCalculatedTissue *tissue: allTissues)
-		tissue->setVisible(prefs.calcalltissues);
+		tissue->setVisible(prefs.calcalltissues && prefs.calcceiling);
```

Comment:
the commit message in **profile: call DiveCalculatedTissue::setVisible() correctly.** confuses me when I look at this change. When you say "pass the correct parameter in the first place"  - is there "another place" where the call to `setVisible()` should be removed entirely because it's no longer needed?

Also, don't have a period at the end of a commit message title.

Reply:
Yes the crazy `setVisible()`, which ignores the visible parameter, is removed in the next commit, a120b5a77e81be08b850ce85fa50fe9d3a2ea37c.

One could squash these commits.